scst-store/amr/architecture.hbs.md

```diff
@@ -6,8 +6,10 @@ This topic describes the architecture of Artifact Metadata Repository (AMR).
 
 ![Diagram of Architecture for AMR Interaction](../images/amr-arch.png)
 
-[//]: # (^ diagram is produced from https://docs.google.com/drawings/d/1IChjcsCL8wcjQ4YG35JNWWy50TGZ1_N-osv-QTBnELg)
-
 AMR Observer consists of managed controllers that watch resources. In Tanzu Application Platform 1.6, AMR Observer watches for ImageVulnerabilityScans and workload ReplicaSets. Workload ReplicaSets are ReplicaSets that contain one container named workload that is produced by the out of the box Supply Chains.
 
-When changes to these resources occur, AMR Observer creates CloudEvents and send these events to Artifact Metadata Repository CloudEvent Handler. Artifact Metadata Repository CloudEvent Handler consumes these CloudEvents and make the required API calls to Metadata Store and Artifact Metadata Repository. 
\ No newline at end of file
+When enabled, the AMR Observer is deployed to the build and run clusters. It starts by communicating with the Kubernetes API Server to obtain the cluster's location ID, which is the GUID of the kube-system namespace. Once the location ID is retrieved, the AMR Observer emits a cloud event, including any operator-defined metadata, to the Artifact Metadata Repository Cloud Event Handler (AMR Cloud Event Handler). This cloud event registration marks that particular location, and subsequent cloud events in the same cluster use the location ID in the source field. This mechanism helps the AMR keep track of observed locations and artifacts.
```

Comment:
`This cloud event registration marks that particular location, and subsequent cloud events in the same cluster use the location ID in the source field.`

This sentence is a bit difficult to read. What do you mean by `mark`?